### PR TITLE
Removed the 'Using Linux' section

### PR DESCRIPTION
**Pull request by aallan**
I don't feel that this section adds anything significant to our documentation, it's all better covered elsewhere. We're not in the business of documenting Linux, the Raspberry Pi documentation should be about Pi-specific topics.

### CODE REVIEW

File: documentation/asciidoc/computers/configuration/raspi-config.adoc [hunk 1]
```diff
@@ -43,7 +43,9 @@ Specify the audio output destination.
 [[change-user-password]]
 ===== Password
 
-The default user on Raspberry Pi OS is `pi` with the password `raspberry`. You can change that here. Read about other xref:using_linux.adoc#linux-users[users].
+You can change the 'default' user password.
+
+NOTE: Until recently the default user on Raspberry Pi OS is `pi` with the password `raspberry`. The default user is now set on first boot using a configuration wizard.
```
Comment (andrum99) [inline documentation/asciidoc/computers/configuration/raspi-config.adoc:48]: `is` -> `was`
`the password` -> `default password`

both in first sentence

Comment (aallan) [reply documentation/asciidoc/computers/configuration/raspi-config.adoc]: Fixed up the "is" to "was". The "the" to "default" isn't good grammar so have kept.

File: documentation/asciidoc/computers/configuration/raspi-config.adoc [hunk 2]
```diff
@@ -43,7 +43,9 @@ Specify the audio output destination.
 [[change-user-password]]
 ===== Password
 
-The default user on Raspberry Pi OS is `pi` with the password `raspberry`. You can change that here. Read about other xref:using_linux.adoc#linux-users[users].
+You can change the 'default' user password.
```
Comment (lurch) [inline documentation/asciidoc/computers/configuration/raspi-config.adoc:46]: There's no longer a default password, so this should probably just say "You can change your user password." ?

File: documentation/asciidoc/computers/remote-access/secure-shell.adoc [hunk 3]
```diff
@@ -31,4 +31,4 @@ Alternatively you can enable it from the terminal using the xref:configuration.a
 
 NOTE: For headless setup, SSH can be enabled by placing a file named `ssh`, without any extension, onto the boot partition of the SD Card. When the Raspberry Pi boots, it looks for the `ssh` file. If it is found, SSH is enabled and the file is deleted. The content of the file does not matter; it could contain text, or nothing at all.
 
-WARNING: When enabling SSH on a Raspberry Pi that may be connected to the internet, you should xref:using_linux.adoc#changing-your-password[change the default password] to ensure that it remains secure.
+WARNING: When enabling SSH on a Raspberry Pi that may be connected to the internet, you should ensure that your password is not easily brute forced.
```
Comment (lurch) [inline documentation/asciidoc/computers/remote-access/secure-shell.adoc:34]: We could link this to `configuration.adoc#change-user-password` ?
And I dunno if "you should ensure that you choose a strong password" would make more sense to the average person than "you should ensure that your password is not easily brute forced" ? :shrug:

**Review by lurch — COMMENTED** [review]
I guess we probably also ought to modify (or remove) the redirect in `documentation/htaccess_extra.txt` ?

**Comment by aallan** [comment]
YOLO

**Pull request merged**